data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml

```diff
@@ -0,0 +1,102 @@
+  This amount of simulation time corresponds to ~12.9 GPU-years on an NVIDIA GeForce GTX 1080Ti.
+  This trajectory dataset with solvent is available at the MolSSI COVID-19 Molecular Structure and Therapeutics Hub: https://covid.molssi.org//org-contributions/#folding--home
+
+  **Solute-only trajectories:**
+  The solute-only trajectories (with counterions) are available as [MDTraj HDF5 files](https://mdtraj.org/1.9.4/hdf5_format.html) that contain both topology and trajectory information.
+  A single trajectory of the WT RBD (RUN3) (~30 MB) can be downloaded using the [AWS CLI](https://aws.amazon.com/cli/):
+  ```
+  aws s3 --no-sign-request cp s3://fah-public-data-covid19-antibodies/vir-collaboration/SARS-CoV-2-ACE2-RBD/munged/solute/PROJ17311/run3-clone0.h5 .
+  ```
+  All HDF5 trajectories (~300 GB) can be retrieved with
+  ```
+  aws s3 --no-sign-request sync s3://fah-public-data-covid19-antibodies/vir-collaboration/SARS-CoV-2-ACE2-RBD/munged/solute/PROJ17311 .
+  ```
+
+  **Entire dataset:**
+  The raw Folding@home dataset is made available through the [AWS Open Data Registry](https://registry.opendata.aws/foldingathome-covid19/) and can be retrieved through the [AWS CLI](https://aws.amazon.com/cli/).
+  The dataset consists of a single project (`PROJ17311`) and has a `RUN*/CLONE*/result*` directory structure.
+  `RUNs` denote different mutants:
+  `RUN0` is RBD N439K;
+  `RUN1` is RBD K417V;
+  `RUN2` is N439K/K417K;
+  `RUN3` is wild-type (WT).
+  `CLONE`s denote different independent replica trajectories.
+
+  To retrieve raw trajectory files in [gromacs XTC format](http://manual.gromacs.org/archive/5.0.4/online/xtc.html)
+  for the whole dataset (7 TB), you can use the [AWS CLI](https://aws.amazon.com/cli/):
+  ```bash
+  aws s3 --no-sign-request sync s3://fah-public-data-covid19-antibodies/vir-collaboration/SARS-CoV-2-ACE2-RBD/raw-data/ .
+  ```
+
+  **Folding@home initial files:** System setup and input files can be downloaded using the [AWS CLI](https://aws.amazon.com/cli/):
+  ```bash
+  aws s3 --no-sign-request sync s3://fah-public-data-covid19-antibodies/vir-collaboration/SARS-CoV-2-ACE2-RBD/setup/ .
+  ```
+
+  **Contributors:** Ivy Zhang, William G. Glass, Tristan I. Croll, Aoife M. Harbison, Elisa Fadda, John D. Chodera.
+
+  **License:** All data is freely available under the Creative Commons [CC0 ("No Rights Reserved")](https://creativecommons.org/share-your-work/public-domain/cc0/) license.
+
+creator: Ivy Zhang
+organization: Folding@home
+lab: Chodera lab
+institute: Memorial Sloan Kettering Cancer Center
+proteins:
+    - spike
+    - RBD
+    - ACE2
+structures:
+    - 6m0j
+    - 1R42
+trajectory: https://fah-public-data-covid19-antibodies.s3.us-east-2.amazonaws.com/vir-collaboration/SARS-CoV-2-ACE2-RBD/munged/solute/PROJ17311/run3-clone0.h5
+size: 2.1 TB
+length: 173.8 µs
+ensemble: NPT
+temperature: 310
+pressure: 1
+solvent: water
+salinity: 0
+forcefields:
```

Comment:
Can you also add `GLYCAM_06j-1` as one of the forcefields used?

Reply:
Done!